README.md

```diff
@@ -255,6 +255,11 @@ $ rustypaste
 
 See [config.toml](./config.toml) for configuration options.
 
+### List endpoint
+
+Set `json_list_enabled` to true in [config.toml](./config.toml) to be able to retrieve a JSON formatted list of files in your uploads directory. This will not include oneshot files, oneshot urls, or urls. 
```

Comment:
Just brainstorming, should we maybe rename this to `list_files` or something similar?

```diff
- json_list_enabled = true
+ list_files = true
```

Also, I think there is no point in prefixing it with `json` since it is a no-brainer to use JSON 😅 

What do you think? @tessus @ajbdev

Reply:
Short answer: yes.

Long answer: If it were to be consistent, it should be `expose_list`. But then I went down the rabbit hole.... What about a var e.g. `expose = ['version', 'list']`? In that case, another deprecation would be necessary. Although using an array to expose endpoints is more future proof. But maybe too much for the scope of this PR. (Just blabbing out thoughts unfiltered.)

Reply:
I like `expose_list`

Reply:
Yup, let's go with `expose_list`. Sounds reasonable.